### ISSUE TITLE
Dragging a node out of the screen causes an error

### ISSUE THREAD

**Issue by zommerberg** [wbkd/react-flow#477]
I am using next.js and when I drag the node out of the screen I get an error:

```Error: Unable to find node on an unmounted component.```

My setup looks like this 
``` <div ref={wrapperRef}>
      <ReactFlow
        onNodeDragStop={onNodeDragStop}
        edgeTypes={{
          custom: CustomEdge,
        }}
        nodeTypes={{
          special: CustomNode,
        }}
        onPaneClick={onPaneClick}
        elements={elements as any}
        onElementClick={onElementClick}
        // onElementsRemove={onElementsRemove}
        onConnect={onConnect}
        onLoad={onLoad}
        connectionLineStyle={{ stroke: '#ddd' }}
        snapToGrid={true}
        snapGrid={[16, 16] as any}
      >
        <Background style={{ background: '#F9F9F9' }} color='#000' gap={16} />
      </ReactFlow>

      <style global jsx>{`
        html,
        body,
        body > div:first-child,
        div#__next,
        div#__next > div,
        div#__next > div > div {
          height: 100%;
        }
      `}</style>
    </div>
```
Is it possible to fix that?
Thanks a lot!

**Comment by moklick** [comment]
That's a bug. As soon as a node leaves the pane we don't render it anymore in order to increase the performance. 

For now the only thing you can do is to set `onlyRenderVisibleNodes` to `false`.

**Comment by moklick** [comment]
Thanks for the report! This is fixed in v5.9.1
